techniques/css/C38.html

```diff
@@ -2,97 +2,89 @@
-                  in the available space and respond to zoom levels.
-               </li>
-               
+                <li>レイアウト領域のサイズを、flexboxプロパティまたは特定のビューポートサイズに対応するメディアクエリを用いて定義することで、
+                 使用可能なスペース内で拡大／縮小／折り返し、かつズームレベルに適応する。
+                </li>
+
+                <li>flexbox コンテナ内の、段落中の単語と同様に適宜折り返したい flexbox 要素と隣接する列として、レイアウト領域を配置する。</li>
+
+               <li>ラベルと入力欄それぞれの width と max-width プロパティを設定することで、
+                 使用可能なスペース内で拡大／縮小、かつズームレベルに適応するようする。
+              </li>
             </ol>
-            <p>All labels and inputs require design finesse by making sure the original size fits
-               the biggest size of the available spaces to achieve good-looking results at a wide
-               range of viewport sizes and zoom levels. For help on flexbox please see the <a href="https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox">MDN article on Flexbox</a>.
+            <p>幅広いビューポートサイズとズームレベルで見た目の良い結果を得るには、
+              全てのラベルと入力欄の元のサイズが利用可能なスペースの中で最大のサイズに収まるようにデザインする。
+              flexbox の詳細については、<a href="https://developer.mozilla.org/ja/docs/Learn/CSS/CSS_layout/Flexbox">MDN の Flexbox についてのページ</a>を参照のこと。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-fitting-labels-inputs-and-flexbox-layout-with-html-and-css.">
-               <h3>Example 1: Fitting labels, inputs and flexbox layout with HTML and CSS.</h3>
-               <p>The following example uses HTML and CSS to fit labels and inputs within various width
-                  containers, including the viewport. The layout regions adjust their size as the viewport
-                  is adjusted. The labels and inputs subsequently adjust their size to fit within the
-                  layout region containers.
-               </p>
-               <p>The zoom level can be increased to 400% without requiring horizontal scrolling. This
-                  particular example uses a percent size for the <code>width</code> and <code>max-width</code> for the labels and inputs. The <code>max-width</code> is applied in order to fix elements spilling out of the grid in a cross-browser way,
-                  as replaced elements such as the <code>select</code> have intrinsic sizing.
-               </p><pre>
+                <h3>事例 1: HTML と CSS を使用した、ラベル、入力欄、及び flexbox レイアウトの配置</h3>
+                <p>次の事例では、HTML と CSS を用いて、ビューポートを含む可変幅のコンテナの中にラベルと入力欄を配置する。
+                  コンテナのサイズの変化に伴いレイアウト領域のサイズも変動し、また、それに伴いラベルと入力欄のサイズもレイアウト領域に収まるように変動する。
+                </p>
+                <p>水平方向のスクロールを必要とすることなくズームレベルを 400% に上げることができる。
+                  この事例ではラベルと入力欄の <code>width</code> と <code>max-width</code> にパーセントサイズを使用している。
+                  また、<code>select</code> のような内在的寸法を持つ要素がグリッドからはみ出るのをクロスブラウザ的に防ぐため <code>max-width</code> を指定している。
```

Comment:
```suggestion
                  また、<code>select</code> のような置換要素は内在寸法を持つため、要素がグリッドからはみ出るのをクロスブラウザ的に防ぐため <code>max-width</code> を指定している。
```

まあHTML LS（わかりやすさのためにMDN）を見る限りC38の記述が胡散臭いことこの上ないですが、
https://developer.mozilla.org/ja/docs/Web/CSS/Replaced_element
いずれにせよreplaced elementsは置換要素と訳されることが多いと思います。